src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -125,8 +133,12 @@ public function trans($message, array $arguments = [], string $domain = null, st
         return $this->getTranslator()->trans($message, $arguments, $domain, $locale);
     }
 
-    public function createTranslatable(string $message, array $parameters = [], string $domain = 'messages'): Translatable
+    public function createTranslatable(string $message, array $parameters = [], string $domain = null): Translatable
     {
+        if (!class_exists(Translatable::class)) {
```

Comment:
it could be an anonymous data object, for general compatibility

Reply:
it could but I doubt this would provide many benefits. The dependency on the component is quite a loose one for someone using the extension.